docs/dev.md

```diff
@@ -96,11 +98,26 @@ If you want to inject other data or remove some data, you can use [`ACH`](https:
 
 ### Get a working Android environment
 
+#### Linux
+
 To be able to build the app for Android, you can follow [this
 guide](https://gist.github.com/drazik/11dfe2014a6b967821df93b9e10353f4) (in
-french for now, will be translated soon, don't hesitate to open a pull
+French for now, don't hesitate to open a pull
 request).
 
+#### macOS
+
+On macOS you can use adoptOpenJDK to have a working Java environment.
+
+See https://github.com/AdoptOpenJDK/homebrew-openjdk
+
+```
+brew cask install adoptopenjdk8
+# Check the adopt open JDK page to see how to install the shell function "jdk"
+jdk 8 # Cordova supports JDK version 1.8 (same as version 8 it seems)
```

Comment:
Don't we want to break line before # Cordova supports ?

Reply:
No the comment is intended for the line "jdk 8".